### PR TITLE
storage: introduce get_aggregated_measures and deprecate get_measures

### PR DESCRIPTION
**Pull request by jd**

### CODE REVIEW

**Comment by jd** [comment]
Fix sorting

File: gnocchi/storage/__init__.py [hunk 1]
```diff
@@ -199,35 +200,55 @@ def _version_check(name, v):
...
         :param to timestamp: The timestamp to get the measure to.
         :param resample: The granularity to resample to.
         """
-        agg_timeseries = utils.parallel_map(
-            self._get_measures_timeserie,
-            ((metric, ag, from_timestamp, to_timestamp)
-             for ag in aggregations))
+        timeseries = self.get_aggregated_measures(
+            metric, aggregations,
+            from_timestamp=None, to_timestamp=None)
```
Comment (chungg) [inline gnocchi/storage/__init__.py]: how come you don't pass in timestamps here?

File: gnocchi/storage/__init__.py [hunk 2]
```diff
@@ -199,35 +200,55 @@ def _version_check(name, v):
...
-                  for ts in aggts]))
-            for aggmethod, aggts
-            in itertools.groupby(agg_timeseries,
-                                 lambda v: v.aggregation.method)
+                   in timeseries[agg].fetch(from_timestamp, to_timestamp)]
+                  for agg in sorted(aggs,
+                                    key=ATTRGETTER_GRANULARITY,
+                                    reverse=True)]))
+            for aggmethod, aggs in itertools.groupby(timeseries.keys(),
+                                                     ATTRGETTER_METHOD)
```
Comment (chungg) [inline gnocchi/storage/__init__.py]: tbh, this is some crazy looking looping but i'm guessing you're going to remove this later on?

File: gnocchi/storage/__init__.py (hunk 1 above)
Comment (jd) [reply gnocchi/storage/__init__.py]: Good catch.

File: gnocchi/storage/__init__.py (hunk 2 above)
Comment (jd) [reply gnocchi/storage/__init__.py]: It's the same as before and that's what the API returns actually. It should be a bit simpler when this method gets entirely removed and moved to the API>

**Comment by jd** [comment]
Fix timestamp passing

File: gnocchi/storage/__init__.py (hunk 2 above)
Comment (chungg) [reply gnocchi/storage/__init__.py]: works for me.

**Pull request merged**